x/auth/legacy/legacytx/amino_signing.go

```diff
@@ -0,0 +1,91 @@
+package legacytx
```

Comment:
Why `legacy/legacytx`? Why not a shorter package name like just `legacy` or `legacy/tx`?

Reply:
because `tx` is overloaded, and `legacy` is too generic. I prefer to avoid package aliasing in import statements. 

But if you prefer to have `legacy` package in each module then OK, I can move the code.

Reply:
`legacy` contains other subpackages, like `v34` , `v3...` 
and if we will put all code directly in `legacy` we will end up aliasing the imports as we do with `types` (`authtypes`, `ktypes` ...)